sdk/iot/hub/inc/az_iot_hub_client.h

```diff
@@ -19,6 +19,18 @@
 
 #include <_az_cfg_prefix.h>
 
+
+/**
+ * @brief Azure IoT service MQTT properties for telemetry publish messages.

```

Comment:
should there be a clarification here that these are MQTT bit fields instead of being our proprietary option values.